docker-compose.yml

```
@@ -16,12 +16,11 @@ services:
       - database
     environment:
       SECRET_KEY: insecure
-      SCHEMA_URL: https://schemas.data.amsterdam.nl/datasets/
+      # In order to use local datasets, run
```

Hier ontbreekt iets, of:

```suggestion
      # In order to use local datasets
```